parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -53,6 +53,9 @@ OS=$(cat /etc/*-release | grep ^ID= | tr -d 'ID="' | awk '{print toupper($0)}')
 UBUNTU_OS_NAME="UBUNTU"
 RHEL_OS_NAME="RHEL"
 COREOS_OS_NAME="COREOS"
+AZURE_SUFFIX=""
+AZURE_STACK_SUFFIX="-azs"
+AZURE_STACK_HYPERKUBE_REPOSITORY="mcr.microsoft.com/azurestack/k8s/core/hyperkube-amd64"
```

Comment:
for now, Please msazurestackdocker/hyperkube-amd64  it will be for current public preview.
And later on, we can move to MCR